ovos_gui_plugin_shell_companion/__init__.py

```diff
@@ -155,11 +158,17 @@ def handle_device_about_page(self, message):
         self.gui['system_info'] = system_information
         self.gui.show_page("SYSTEM_AdditionalSettings.qml", override_idle=True)
 
-    def handle_display_wallpaper_rotation_config_set(self, message):
+    def handle_display_wallpaper_rotation_config_set(self, message: GUIMessage):
+        """
+        Handle a GUI event requesting a new wallpaper rotation setting
+        @param message: `speaker.extension.display.set.wallpaper.rotation`
+        """
         wallpaper_rotation = message.data.get("wallpaper_rotation", False)
         self.local_display_config["wallpaper_rotation"] = wallpaper_rotation
         self.local_display_config.store()
```

Comment:
this setting also doesn't need to be saved, the source of truth for this is tracked by wallpaper plugin. this whole handler can be dropped

relates to #26

Reply:
The setting does need to be referenced for the settings page to render properly, but it should come from the wallpaper plugin and not a separate config store

Reply:
Looks like this is already happening in QML 

https://github.com/OpenVoiceOS/ovos-gui-plugin-shell-companion/blob/feacd67d7931b2603d21131b6e50e3cff15a3d61/ovos_gui_plugin_shell_companion/res/ui/settings/display_settings.qml#L50

I'll see if that's already working